src/style.css

```diff
@@ -0,0 +1,124 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-weight: 300;
+  padding: 0;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.icon {
+  position: absolute;
+  bottom: 0;
+  left: 26px;
+}
+
+.iconSmall {
+  position: absolute;
+  bottom: 0;
+  left: 11px;
+  top: 5px;
+  height: 11px;
+  width: 11px;
+}
+
+.form-field {
+  margin-top: 20px;
+  width: 100%;
+}
+
+.form-fieldSmall {
+  margin-top: 20px;
+  width: 100%;
+}
+
+.form_input {
+  font-family: inherit;
+  height: 70px;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.bigInput {
+  position: relative;
+  width: 100%;
+}
+
+.bigInput:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigInput:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.bigInput::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.smallInput {
+  position: relative;
+  width: 100%;
+}
+
+.form_input--small {
+  font-family: inherit;
+  height: 42px;
+  width: 100%;
+  font-size: 14px;
+  line-height: 19px;
+  padding-left: 33px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form_input--small:hover {
+  background: ffffff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.form_input--small:focus {
+  background: linear-gradient(180deg, ffffff 0%, #f6f6f7 100%);
```

Comment:
The background color is missing a '#' before the color code. It should be `#ffffff`.